next-tavla/src/MyBoards/scenarios/Row/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { IconButton } from '@entur/button'
+import { CopyIcon, EditIcon } from '@entur/icons'
+import { TSettings } from 'types/settings'
+import { useToast } from '@entur/alert'
+import { useRouter } from 'next/router'
+import classes from './styles.module.css'
+
+function Row({
+    board,
+}: {
+    board: { id: string; settings: TSettings | undefined }
+}) {
+    const link = `https://lite-tavla.web.app/${board.id}`
```

Comment:
Denne burde bruke linken fra `window` eller useRouter

Reply:
router sin basepath returnerer alltid tom string. hvis man gjør det med window uten useeffect, får man "window is not defined" feil